plugins/scaffolder/src/components/TemplatePage/TemplatePage.tsx

```diff
@@ -135,6 +135,8 @@ export const TemplatePage = () => {
               onReset={handleFormReset}
               onFinish={handleCreate}
               steps={schema.steps.map(step => {
+                step.schema.type = step.schema.type || 'object';
```

Comment:
This has moved over to the `transformSchemaToProps` function in the `MultistepJsonForm` component instead

Reply:
I didn't get you here, do you mean this has to be move to the `transformSchemaToProps` or it is already there?

Reply:
Yeah it has to be moved, purpose of the comment was to help resolve the pending merge conflict 😁